tests/test_comm.rs

```diff
@@ -1,105 +1,82 @@
 use common::util::*;
-use std::ffi::OsStr;
 
 static UTIL_NAME: &'static str = "comm";
-fn at_and_ucmd() -> (AtPath, UCommand) {
-    let ts = TestScenario::new(UTIL_NAME);
-    let ucmd = ts.ucmd();
-    (ts.fixtures, ucmd)
-}
 fn new_ucmd() -> UCommand {
     TestScenario::new(UTIL_NAME).ucmd()
 }
```

Comment:
Cool!

In addition, if the macro is only used in this project, I reckon we can get the name using macro module_path!().
Because the path of test module is something like `tests::test_<util>`, so the name is `&module_path!()[12..]`